qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/RouteLineFeaturesActivity.kt

```diff
@@ -0,0 +1,248 @@
+            .alternativeRouteTrafficLineScaleExpression(routeTrafficLineScaleExpression)
+            .build()
+    }
+
+    private val routeLineOptions: MapboxRouteLineOptions by lazy {
+        MapboxRouteLineOptions.Builder(this)
+            .withRouteLineResources(routeLineResources)
+            .withRouteLineBelowLayerId("road-label-navigation")
+            .withRouteStyleDescriptors(
+                listOf(
+                    RouteStyleDescriptor("altRoute1", Color.CYAN, Color.MAGENTA),
+                )
+            )
+            .build()
+    }
+
+    private val routeLineView by lazy {
+        MapboxRouteLineView(routeLineOptions)
+    }
+
+    private val routeLineApi: MapboxRouteLineApi by lazy {
+        MapboxRouteLineApi(routeLineOptions)
+    }
+    private var showTraffic = true
+    private var showPrimaryRoute = true
+    private var showAltRoutes = true
+    private var routesAreShowing = true
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(viewBinding.root)
+        initStyle()
+        initListeners()
+    }
+
+    private fun initListeners() {
+        viewBinding.btnToggleTrafficVisibility.setOnClickListener {
+            when (showTraffic) {
+                true -> routeLineView.hideTraffic(mapboxMap.getStyle()!!)
+                false -> routeLineView.showTraffic(mapboxMap.getStyle()!!)
+            }
+            showTraffic = !showTraffic
+        }
+
+        viewBinding.btnTogglePrimaryRouteVisibility.setOnClickListener {
+            when (showPrimaryRoute) {
+                true -> routeLineView.hidePrimaryRoute(mapboxMap.getStyle()!!)
+                false -> routeLineView.showPrimaryRoute(mapboxMap.getStyle()!!)
+            }
+            showPrimaryRoute = !showPrimaryRoute
+        }
+
+        viewBinding.btnToggleAlternativeRouteVisibility.setOnClickListener {
+            when (showAltRoutes) {
+                true -> routeLineView.hideAlternativeRoutes(mapboxMap.getStyle()!!)
+                false -> routeLineView.showAlternativeRoutes(mapboxMap.getStyle()!!)
+            }
+            showAltRoutes = !showAltRoutes
+        }
+
+        viewBinding.btnClear.setOnClickListener {
+            when (routesAreShowing) {
+                true -> {
+                    routeLineApi.clearRouteLine {
+                        routeLineView.renderClearRouteLineValue(mapboxMap.getStyle()!!, it)
+                    }
+                    viewBinding.btnClear.text = "Draw Routes"
+                }
+                false -> {
+                    drawRoutes()
+                    viewBinding.btnClear.text = "Clear Routes"
+                }
+            }
+            routesAreShowing = !routesAreShowing
+        }
+    }
+
+    @SuppressLint("MissingPermission")
+    private fun initStyle() {
+        mapboxMap.loadStyleUri(
+            "mapbox://styles/lukaspaczos/ckirf03jn7hv817nrr69ndwdw",
```

Comment:
Let's just use a default navigation style - `NavigationStyles.NAVIGATION_DAY_STYLE`.

Reply:
I"ll change it but that style has a lot of colors because of the traffic indicators and for this activity I colored the route lines so that it's easier to see the different parts of the lines as well as differentiation between the alternative routes.

Reply:
The `LIGHT` style could work as well if we want to keep it simple.